spec/acceptance/storage_account_spec.rb

```diff
@@ -1,10 +1,12 @@
 require 'spec_helper_acceptance'
 
-describe 'azure_storage_account when creating a storage account' do
+describe 'azure_storage_account when creating a storage account', :focus => true do
   include_context 'with a known name and storage account name'
   include_context 'destroy left-over created ARM resources after use'
 
   before(:all) do
+    @client = AzureARMHelper.new
+    @name = @client.get_simple_name(@name)
```

Comment:
hmm. The function for `get_simple_name` is in the spec_helper_acceptance, but it's not called anywhere...